app/mailers/startup_mailer.rb

```diff
@@ -1,11 +1,21 @@
 # Mails sent out to teams, as a whole.
 class StartupMailer < SchoolMailer
   def feedback_as_email(startup_feedback)
+    emails_fullnames = startup_feedback.timeline_event.founders.map { |e| {"#{e.fullname} <#{e.email}>" => {"fullname" => e.fullname, "email" => e.email}} }
+    emails_fullnames.each do | student |
```

Comment:
you could save a loop by doing this work in the `map` call

Reply:
i think you can also destructure the arguments and pull out `send_to, values` without having to do a nested `each` loop

Reply:
Could you please check my new commit?